CHANGELOG-LATEST.md

```diff
@@ -1,18 +1,36 @@
+## 1.0.0-beta.3 API Changes
+
+This latest release updates the Android SDK dependency from v7 to [v8](https://github.com/RevenueCat/purchases-android/releases/tag/6.0.0) to use BillingClient 7 and updates the iOS SDK dependency from v4 to v5 to use StoreKit 2 by default in the SDK.
+
+### Migration Guides
+
+- For a detailed guide to all of the changes, please see the [KMP V1.0.0-beta.3 Migration Guide](https://github.com/RevenueCat/purchases-kmp/blob/main/migrations/1.0.0-beta.3-MIGRATION.md).
+- See [Android Native - V8 API Migration Guide](https://github.com/RevenueCat/purchases-android/blob/main/migrations/v8-MIGRATION.md) for a more thorough explanation of the Android changes.
+- See [iOS Native - V5 Migration Guide](https://github.com/RevenueCat/purchases-ios/blob/main/Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md) for a more thorough explanation of the iOS changes. Notably, this version uses StoreKit 2 to process purchases by default.
+
+### New Minimum OS Versions
+
+This release raises the minimum required OS versions to the following:
+
+- iOS 13.0
+- tvOS 13.0
+- watchOS 6.2
+- macOS 10.15
```

Comment:
We do not ([yet](https://github.com/RevenueCat/purchases-kmp/issues/126)) support Apple targets other than iOS. See also https://github.com/RevenueCat/purchases-kmp/pull/153. This comment also applies to `CHANGELOG.md`.

Reply:
@JayShortway Fixed in https://github.com/RevenueCat/purchases-kmp/pull/152/commits/75e1600b75a5d26141ab0a8c8803a467b7d47bcb :D

Reply:
Nice 🙌